python/doc/examples/reliability_sensitivity/axial_stressed_beam.ipynb

```diff
@@ -0,0 +1,885 @@
u82bzvXHXXXDggL3xiauowJLo98EH0L69GTC06JyKHj1g1Sq48EJ7YxOR6HPRRfDtt3DuuWa5aA7Ddu0CVxuLhJkKLIle27bBFVeYyZl/+cWsS0w0AwtmZUHTpvbGJyLRKz3dXGH45JPmewPM90jv3uZ7Zft2e+MTx1OBJdFn/34YMwZatYLp0wPrzz7bXIZ9550Qp11XRI4gLs58X3zzjfn+KDJ9uvl+efhh830jEgb6KyXRIy/PjM7cujVkZJjzrsCMcfOf/5gTVdu0sTVEEYlBbdqY74///CcwZtb+/fDQQ+b75oUXzPePSAipwBL7+Xzw3HPQsiXceKPpGgRISDBT3axZY+Ya01ErEamquDjzPbJmjfleKbowZts2873TsiVMnGi+j0RCQH+xxD47d8Ijj5gBAkeONFf7FPnzn+G778xkzamptoUoIg6Tmmq+V77/3nzPFNm8GW6+2XwfPfJIYABjkSpSgSWRZVnw6acwYgQceyw8+CDs2BH4/QUXwBdfwNtva9BQEQmfE04w3zNffGEmii+yY4f5Xjr2WPM99dlnGhFeqkQFlkTGpk0wbpw5F6JbN5gyJXAoPi4OLr4Yli+H//4XOne2NVQRcZHOneHdd833z8UXB05F8PnM91TXrnDiieb7a9MmW0OV2KICS8Jn/Xp4+mno0gWaNTMjLK9ZE/h97drmCp+ff4a33jKTtIqI2OG008z30M8/wx13mO+nIj/9ZL6/mjUz32dPP22+30QOI8HuAMRBDhwwh9Pnz4d582D16rLv1707XHONGQywRo2IhigicljHHWcGJR0zxowIP3kyLFgQ+P3SpeZ2xx3myFa/ftCnD5x5JtSsaVfUEoVUYEnV/fqrOaz+5ZfmC2jp0vIvdT75ZBg8GIYOhRYtIhqmiEil1agBV11lbj//DDNmmLkNV60K3Gf1anMbP95cldixI5x1ljkNomtXaNjQvvjFdiqw5Mgsi6Tff8fzwQewcqUpqpYvD8wJWJa4OPMfXf/+5rwGnbAuIrGqRQt44AFz+/FHMwfq3Lnmn8rCQnOfgoLAd+Mzz5h1rVqZrsd27cx0XyecELi/OJ6rCqznnnuOJ598ku3bt9O2bVsmTJjAWWedZXdY0aGwEHbtMidxrllT4pawZg199u498nO0aGG6/3r0MNPb1KsX9rBFRCKqTRu4/35z273bzJW6cCEsWWKGfgi2dq25HeIF+lerRlz79qboOv540yXZvLn52bixxvtzENcUWK+//jqjRo3iueeeo2vXrrzwwgv07duXH374gWOPPdbu8MLDsiAnxxROu3eX/LltmzkCtXVr4Gc53XueslampJj/zDp1Mj//9Ccz95eIiFvUq2dOexg61Cz/9ps5D3XJEnNbtqzU92rCwYNm/bJlpZ8vMdGcSN+0KTRqZLoYg382aGBGoq9d23wHJ7jmT3hMcs3Wefrpp7n22mv5y1/+AsCECRN4//33mThxIo8//njkA/L5TLHj85lbXl7Z7bKWDx6Efftg797Arazl7GzIzz+6OOPisI47jp116pB61lnEd+5siqqWLfWflohIsLp1zVh+F1xglv1+0xOwahV89x2F33zDgWXLqPHrr2U/Pi+v1NL/86MAABDISURBVFGvw6pZ0xRbtWtDnTqBdkoKVK8O1apBcnLJW/C6atVMUef1mltRu+hnaiokJYXmvXEhVxRYeXl5rFixgnvvvbfE+t69e/PZZ5/ZE9SCBebKk2hwzDHmP6aiW8uW5lyB1q3h+OPJj4tj6bx59OvXj3iv1+5oRURig9cLbduaG1Dg9/PhvHn0O/tsvOvXw8aNZriHDRtK/qzoBNT79pnb1q3hif+DD6BXr/A8twu4osDatWsXBQUFNPzDFR0NGzZkR/Ao4kF8Ph++oDmpcnJyAPD7/fj9/uL1Re3gdRXhiY8Py5tvJSdDrVrmP5tatbBSU6FePfOzbl1ITcWqWxcaNcJq0gSaNDH/6RxGVXOMFU7PD5yfo9PzA+XoBMX5JSWZk97bty99J8syRdOvv+L59dcSP/nf//Ds2WNO/cjOxpOdbXoqsrPxVLQoq4T8uDisSm6L8rahU7fp4Xgsy/lzAGzbto0mTZrw2Wef0aVLl+L1jz76KK+++io//vhjqcdkZGQwZsyYUuszMzOpfoSCpCJqbtnCidOmUej1UpiQYG5Fba83cDu0XOD1YiUkUHBofX61auQnJwdu1apRkJyMVTSBqYiIuIanoICE/fvx5uaSkJtLfF4e8Xl5xB36GXyLy8sj3ucj3u/Hk59PXEEBnoIC4vx+087PJy4/n9WXX86+EJ1bm5uby7Bhw8jOziYlJSUkzxntXHEEKzU1lfj4+FJHq3bu3FnqqFaR++67j9GjRxcv5+TkkJ6eTu/evUvsHH6/n6ysLHr16oW3st1n119fufvb5KhyjAFOzw+cn6PT8wPl6ASxll/9KjymvByLeoHcxBUFVmJiIqeddhpZWVlceOGFxeuzsrIYOHBgmY9JSkoiqYyT+7xeb5kfjPLWO4nTc3R6fuD8HJ2eHyhHJ3B6flA6R6fnWxZXFFgAo0eP5sorr6RTp0506dKFSZMmsWnTJm688Ua7QxMRERGHcU2Bddlll7F7924efvhhtm/fTrt27Zg3bx7NmjWzOzQRERFxGNcUWAA333wzN998s91hiIiIiMNppEgRERGREFOBJSIiIhJiKrBEREREQkwFloiIiEiIqcASERERCTEVWCIiIiIh5qphGo5G0ZSNfxzu3+/3k5ubS05OjmNHqnV6jk7PD5yfo9PzA+XoBE7PD8rPsehvpwumPy6mAquC9u7dC0B6iCa+FBERcZu9e/dSu3Ztu8OICI/lpnLyKBQWFrJt2zZq1aqFx+MpXl80CfTmzZsdO0O403N0en7g/Bydnh8oRydwen5Qfo6WZbF3717S0tKIi3PH2Uk6glVBcXFxNG3atNzfp6SkOPYDU8TpOTo9P3B+jk7PD5SjEzg
```

Comment:
Github suggestion failure:
The key point is to define the importance distribution in the U-space. To define it, we use a multivariate standard Gaussian and configure it so that the center is equal to the design point in the U-space.

Reply:
Ok. Fixed.